e2e-polybft/framework/test-cluster.go

```diff
@@ -594,20 +620,25 @@ func NewTestCluster(t *testing.T, validatorsCount int, opts ...ClusterOption) *T
 	for i := 1; i <= int(cluster.Config.ValidatorSetSize); i++ {
 		dir := cluster.Config.ValidatorPrefix + strconv.Itoa(i)
 		cluster.InitTestServer(t, dir, cluster.Bridge.JSONRPCAddr(),
-			true, !cluster.Config.WithoutBridge && i == 1 /* relayer */)
+			true, !cluster.Config.WithoutBridge && i == 1 /* relayer */, false /* byzantine */)
 	}
 
 	for i := 1; i <= cluster.Config.NonValidatorCount; i++ {
 		dir := nonValidatorPrefix + strconv.Itoa(i)
 		cluster.InitTestServer(t, dir, cluster.Bridge.JSONRPCAddr(),
-			false, false /* relayer */)
+			false, false /* relayer */, false /* byzantine */)
+	}
+
+	for i := 1; i <= int(cluster.Config.ByzantineNodeCount); i++ {
+		dir := byzantineNodePrefix + strconv.Itoa(i)
+		cluster.InitTestServer(t, dir, cluster.Bridge.JSONRPCAddr(), false, false /* relayer */, true /* byzantine */)
 	}
 
 	return cluster
 }
 
 func (c *TestCluster) InitTestServer(t *testing.T,
-	dataDir string, bridgeJSONRPC string, isValidator bool, relayer bool) {
+	dataDir string, bridgeJSONRPC string, isValidator bool, relayer bool, byzantine bool) {
```

Comment:
It would be nicer if we could have `isValidator`, `relayer` and `byzantine` represented as flags enum. It was cumbersome before, but it gets worse with each new boolean parameter. WDYT?

Reply:
Maybe we can modify the `InitTestServer` to accept initial `TestServerConfig` instead of all those parameters?

Reply:
The task for this is now in the backlog.
https://polygon.atlassian.net/browse/EVM-781